setup.py

```diff
@@ -858,7 +858,7 @@ def debug_build():
         project_urls={
             "Documentation": "https://pillow.readthedocs.io",
             "Source": "https://github.com/python-pillow/Pillow",
-            "Funding": "https://tidelift.com/subscription/pkg/pypi-pillow",
+            "Funding": "https://tidelift.com/subscription/pkg/pypi-pillow?utm_source=pypi-pillow&utm_medium=pypi",
```

Comment:
Something like this should fix the lint:
```suggestion
            "Funding": "https://tidelift.com/subscription/pkg/pypi-pillow?"
            "utm_source=pypi-pillow&utm_medium=pypi",
```

Reply:
Thanks, accepted